deploy/packer/base.json

```diff
@@ -44,6 +44,12 @@
     {
       "type":           "shell",
       "inline":         ["sudo unattended-upgrade"]
+    },
+    {
+      "type": "shell",
```

Comment:
I definitely like that packer allows us to inline this here, but could we maybe move this to a script where comments can be added as to why we are doing this in the first place?  Just so that if anyone later on looks at this, they know in older kernels to not remove it.